yasser_sharabati/tasks.json

```diff
@@ -0,0 +1,21 @@
+{
+    "tasks": [
+        {
+            "task_id": 1,
+            "title": "yasser",
+            "description": "yasser",
+            "due_date": "2025-02-20",
+            "status": "InProgress"
+        }
+    ],
+    "urgent_tasks": [
+        {
```

Comment:
You can store both of them as tasks.
For tasks, you can store the priority and status as null values.

When retrieving the values from this file, you can store them in either tasks or urgent tasks based on those values

Reply:
I see if I do that this will not efficient be because I add another attribute while I can do not it